src/utils/api/contact-mfa.ts

```diff
@@ -24,12 +26,11 @@ export function deserializeContactMfa<T>(data: T): T {
 export async function createContactMfa(
   contactId: string,
   dataIn: CreateContactMfaData
-): Promise<CreateContactMfaData> {
-  const { data } = await axios.post<Serial<CreateContactMfaData>>(
+): Promise<void> {
+  await axios.post<Serial<CreateContactMfaData>>(
```

Comment:
You can remove the `Serial<CreateContactMfaData>>` here as that's the returned data type which in this case is empty

Reply:
Check